### DISCUSSION TITLE
Make NewSkins From Stepmania 5.1.-3 Usable

### DISCUSSION THREAD

**Discussion by TRRESTGHGYTCFTGBCXSZD** [TeamRizu/OutFox#317] [03 - Feature Requests]
Newskins Are Located at (Outfox Root Folder)/Appearance/Noteskins/NewSkin, But Do not change Newskin to Newskin, This can cause Instabilities

**Comment by Scraticus** [comment]
The methodolody used in 5.2, which is what this build actually is does not make it compatible with outfox. There is no plan currently to change this.

**Reply by TRRESTGHGYTCFTGBCXSZD** [reply]
just add as part of code

**Reply by Gabrimax98** [reply]
I think it unfortunately isn't as easy as just adding it as "part of the code". The amount of differences of code between Outfox and 5.2 (even between 5.2 and 5.1) is so vast it would probably be faster, and easier, to convert the skin you intend to use to a standard 5.1 Noteskin type than for the team to properly implement the whole Newskin situation, with proper code modernizing and bug fixing so it won't conflict with the rest of the newer, more modern engine.

So, if you **absolutely** need to have that one Newskin you're using, then maybe somebody in our Discord server can help you out in figuring what to do to convert it to a Noteskin.
If it's the feature you need then yeah, @Scraticus already said what the issues are.
